wherehows-web/tests/integration/components/datasets/containers/data-systems-count-test.ts

```diff
@@ -2,21 +2,11 @@ import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
 import { render } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
-
-import { startMirage } from 'wherehows-web/initializers/ember-cli-mirage';
 import { nonHdfsUrn } from 'wherehows-web/mirage/fixtures/urn';
 
 module('Integration | Component | datasets/containers/data-systems-count', function(hooks) {
   setupRenderingTest(hooks);
 
-  hooks.beforeEach(function(this: any) {
-    this.server = startMirage();
```

Comment:
Why was this removed? The container will use the mirage config to fetch data
https://www.ember-cli-mirage.com/docs/v0.4.x/manually-starting-mirage/
Is this still the case with this change, you can verify by breakpointing the component task to see if the data is actually returned. It is possible that the test may show up as passed but actually it does not really execute correctly because the fetch operation returns a default value : - )

Reply:
Well, you either autostart mirage or use this. Booth we will have the problem of booting pretender twice.